pkg/webui/console/components/webhook-form/index.js

```diff
@@ -225,14 +238,21 @@ export default class WebhookForm extends Component {
             disabled={update}
           />
           <WebhookFormatSelector name="format" required />
+          {reactivate && (
+            <React.Fragment>
+              <div>
+                <Form.SubTitle title={reactivateWebhookTitle} />
+                {reactivateButton}
```

Comment:
There is no reason to pass the button as a prop. You can determine whether the button is necessary from the available props:

```javascript
const mayReactivate = update &&
  initialWebhookValue &&
  initialWebhookValue.health_status &&
  initialWebhookValue.health_status.healthy
```

and then you can render the button using new handler props (similar to how submitting and deleting is currently handled):

```
<Form.SubTitle title={m.reactivateWebhookTitle} />
<Button
  message={m.reactivateButton}
  icon={'refresh'}
  onClick={handleReactivate}
  className={style.activateWebhookButton}
/>
```

Reply:
To do this I need to map the `health_status` value into the form `initialvalues`. Wouldn't it defeat the purpose of decoupling the reactivate button from the form? Because then when submitting the form health_status will always also be included